deployments/arbitrum/usdc/migrations/1679020486_configurate_and_ens.ts

```diff
@@ -0,0 +1,355 @@
+          usdcGasParams.maxFeePerGas,               // uint256 _gasPriceBid,
+          utils.defaultAbiCoder.encode(
+            ['uint256', 'bytes'],
+            [usdcGasParams.maxSubmissionCost, '0x']
+          )                                         // bytes calldata _data
+        ],
+        value: usdcGasParams.deposit
+      },
+      // 4. Approve the COMP gateway to take Timelock's COMP for bridging
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [compGatewayAddress, COMPAmountToBridge]
+      },
+      // 5. Bridge COMP from mainnet to Arbitrum rewards
+      {
+        contract: l1GatewayRouter,
+        signature: 'outboundTransferCustomRefund(address,address,address,uint256,uint256,uint256,bytes)',
+        args: [
+          COMP.address,                             // address _token,
+          refundAddress,                            // address _refundTo,
+          rewards.address,                          // address _to,
+          COMPAmountToBridge,                       // uint256 _amount,
+          compGasParams.gasLimit,                   // uint256 _maxGas,
+          compGasParams.maxFeePerGas,               // uint256 _gasPriceBid,
+          utils.defaultAbiCoder.encode(
+            ['uint256', 'bytes'],
+            [compGasParams.maxSubmissionCost, '0x']
+          )                                         // bytes calldata _data
+        ],
+        value: compGasParams.deposit
+      },
+      // 6. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(updatedMarkets)]
+        )
+      },
+    ];
+
+    const description = 'XXX'; // XXX add description
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(mainnetActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async verify(deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) {
+    const ethers = deploymentManager.hre.ethers;
+
+    const {
+      comet,
+      rewards,
+      WBTC,
+      WETH,
+      ARB
+    } = await deploymentManager.getContracts();
+
+    // 1.
+    const wbtcInfo = await comet.getAssetInfoByAddress(WBTC.address);
+    const wethInfo = await comet.getAssetInfoByAddress(WETH.address);
+    const arbInfo = await comet.getAssetInfoByAddress(ARB.address);
+    expect(wbtcInfo.supplyCap).to.be.eq(exp(400, 8));
+    expect(wethInfo.supplyCap).to.be.eq(exp(11_000, 18));
+    expect(arbInfo.supplyCap).to.be.eq(exp(10_000_000, 18));
+    // XXX
+    // expect(await comet.pauseGuardian()).to.be.eq('');
```

Comment:
Let's use the state differ here to have an exact check of all the params that have changed. See example: https://github.com/compound-finance/comet/blob/c91bffce984ec6f8276b6f7e9ef362d9293416c1/deployments/optimism-goerli/usdc/migrations/1679592519_configurate_and_ens.ts#L158-L174

You might need to carry over a few lines of changes to `DiffState.ts` from the Optimism branch.

Reply:
updated `DiffState.ts` and added a diffState-based test in place of these checks.

i've commented it out for the time-being; will uncomment it once we've deployed the contracts